src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -14,27 +14,46 @@
 			@reset-filters="resetFilters"
 			@handle-overlay="handleQuickFiltersOverlay"
 		/>
+		<div
+			class="tw-flex tw-flex-wrap"
+		>
+			<kiva-classic-basic-loan-card
+				v-for="(loan, index) in loans"
+				:key="`loan-${loan.id}`"
+				:item-index="index"
+				:loan-id="loan.id"
+				:exp-label="expLabel"
```

Comment:
`expLabel` doesn't exist on this component, so there's an error in the console for `/lending-home`